.github/workflows/wheels.yml

```diff
@@ -0,0 +1,215 @@
+        with:
+          python-version: 3.8
+
+      - name: Build sdist
+        shell: bash
+        run: |
+          cd python
+          python setup.py sdist
+
+      - name: Upload sdist
+        uses: actions/upload-artifact@v2
+        with:
+          name: sdist
+          path: python/dist
+
+      - name: Build wheels in docker
+        shell: bash
+        run: |
+          docker run --rm -v `pwd`:/project -w /project quay.io/pypa/manylinux2010_x86_64 bash docker/buildwheel.sh
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Run tests (3.8)
+        run: |
+          python -VV
+          # We install in this odd way to make sure we get both deps and a local kastore
+          pip install kastore --only-binary kastore -f python/dist/wheelhouse
+          pip uninstall -y kastore
+          pip install -v kastore --only-binary kastore -f python/dist/wheelhouse --no-index
+          python -c "import kastore"
+          pip install -r python/requirements/CI/tests/requirements.txt
+          python -m nose -v python
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Run tests (3.7)
+        run: |
+          python -VV
+          # We install in this odd way to make sure we get both deps and a local kastore
+          pip install kastore --only-binary kastore -f python/dist/wheelhouse
+          pip uninstall -y kastore
+          pip install -v kastore --only-binary kastore -f python/dist/wheelhouse --no-index
+          python -c "import kastore"
+          pip install -r python/requirements/CI/tests/requirements.txt
+          python -m nose -v python
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Run tests (3.6)
+        run: |
+          python -VV
+          # We install in this odd way to make sure we get both deps and a local kastore
+          pip install kastore --only-binary kastore -f python/dist/wheelhouse
+          pip uninstall -y kastore
+          pip install -v kastore --only-binary kastore -f python/dist/wheelhouse --no-index
+          python -c "import kastore"
+          pip install -r python/requirements/CI/tests/requirements.txt
+          python -m nose -v python
+
+      - name: Upload Wheels
+        uses: actions/upload-artifact@v2
+        with:
+          name: linux-wheels
+          path: python/dist/wheelhouse
+
+
+  PyPI_Upload:
+    runs-on: ubuntu-latest
+    needs: ['windows', 'OSX', 'manylinux']
+    steps:
+      - name: Download all
+        uses: actions/download-artifact@v2
+      - name: Move to dist
+        run: |
+          mkdir dist
+          cp */*.{whl,gz} dist/.
+      - name: Publish distribution to Test PyPI
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/py')
```

Comment:
I don't think we can have ``py_`` tags see https://github.com/tskit-dev/kastore/issues/73 and https://github.com/tskit-dev/tskit/issues/74

I think this is because RTD expects tags that are PEP440 version strings.

So, Python versions are tagged with the version number, and C versions are tagged "C_x.y.z". We can test for this here with a tag that does not start with "C_".

Reply:
Yep, saw the note in #110 happy to fix #73 here but I'll leave it open till we have a release that complies.